src/main/java/com/fasterxml/jackson/databind/MapperFeature.java

```diff
@@ -529,6 +529,38 @@ public enum MapperFeature implements ConfigFeature
      */
     USE_STD_BEAN_NAMING(false),
 
+    /**
+     * Feature that determines how to handle property names with mixed capitalization patterns.
+     * This feature specifically addresses cases where getter/setter methods have names that
+     * start with uppercase letters followed by more uppercase letters.
+     * <p>
+     * When enabled, the following rules apply:
+     * <ul>
+     *   <li>For single-letter uppercase prefixes followed by uppercase (like "getIPhone"):
+     *       the first letter is lowercased ("iPhone")</li>
+     *   <li>For multiple uppercase letters at start (like "getKBSBroadCasting" or "getURL"):
+     *       the original case is preserved ("KBSBroadCasting", "URL")</li>
+     * </ul>
+     * <p>
+     * When enabled, overwrites {@link #USE_STD_BEAN_NAMING}.
+     * When disabled, Jackson's standard property naming rules apply using {@link #USE_STD_BEAN_NAMING}.
+     * <p>
+     * Examples with this feature enabled:
+     * <ul>
+     *   <li>"getIPhone" → "iPhone" (single uppercase + uppercase)</li>
+     *   <li>"getDLogHeader" → "dLogHeader" (single uppercase + uppercase)</li>
+     *   <li>"getKBSBroadCasting" → "KBSBroadCasting" (multiple uppercase)</li>
+     *   <li>"getURL" → "URL" (multiple uppercase)</li>
+     *   <li>"getValue" → "value" (standard case, no special handling)</li>
+     * </ul>
+     * <p>
+     * This feature is disabled by default for backward compatibility in Jackson 2.x version and
+     *  will be enabled by default in Jackson 3.0 version.
```

Comment:
Needs discussion

Surprisingly setting default to `true` does not break anything so I figured enable by default in Jackson 3.